pretraining/transforms.py

```diff
@@ -0,0 +1,121 @@
+
+import numpy as np
+import monai.transforms as transforms
+
+
+def train_transforms(crop_size, patch_size, device="cuda", type="pretraining"):
+
+    if type == "pretraining":
+        # NOTE: the pre-trainining is done for SC segmentation
+        all_keys = ["image", "label_sc"]
+        
+        monai_transforms = [
+            # pre-processing    
+            transforms.LoadImaged(keys=all_keys, image_only=False),
+            transforms.EnsureChannelFirstd(keys=all_keys),
+            transforms.Orientationd(keys=all_keys, axcodes="RPI"),
+            # NOTE: spine interpolation with order=2 is spline, order=1 is linear
+            transforms.Spacingd(keys=all_keys, pixdim=(1.0, 1.0, 1.0), mode=(2, 1, 1)),
```

Comment:
`mode=(2, 1, 1)` --> `mode=(2, 1)` since we have only two keys:

```suggestion
            transforms.Spacingd(keys=all_keys, pixdim=(1.0, 1.0, 1.0), mode=(2, 1)),
```

Reply:
Nice catch! Done in commit https://github.com/ivadomed/model-seg-dcm/pull/11/commits/c26da42ec393b6406a92cc679dc6fb20ac265d5b

(wanted to fix both the issues in a single commit to avoid redundant commits hence didn't accept commit suggestions)